xbmc/input/IRTranslator.h

```diff
@@ -31,6 +32,7 @@ class CIRTranslator
 
   unsigned int TranslateButton(const std::string &szDevice, const std::string &szButton);
 
+  static uint32_t TranslateButton(const TiXmlElement *pButton);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types